boards/shields/lcd_par_s035/lcd_par_s035_8080.overlay

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/dt-bindings/mipi_dbi/mipi_dbi.h>
+
+/{
+	chosen {
+		zephyr,display = &st7796s;
+	};
+};
+
+&zephyr_flexio_lcd {
+	status = "okay";
+	#address-cells = <1>;
+	#size-cells = <0>;
+	st7796s: st7796s@0 {
+		compatible = "sitronix,st7796s";
+		reg = <0>;
+		mipi-max-frequency = <43000000>;
```

Comment:
Is the clock frequency of this display 43MHz? From the comment in the board DTS, it seems the display clock should be set to 1MHz, since the actual clock line for the interface is running at 1 MHz (given 16 data lines and a baud rate of 16 Mbps)

Reply:
I have deleted this as this is not used by the driver